index.js

```diff
@@ -70,14 +71,29 @@ const dump = function ({
     args.push("--format");
     args.push(format);
   }
-  return execa(pgDumpPath, args, {});
+  if (verbose) {
+    args.push("--verbose");
+  }
+
+  const subprocess = execa(pgDumpPath, args, {});
+
+  subprocess.stdout.on('data', (data) => {
+    console.log(data.toString().trim());
+  });
+
+  subprocess.stderr.on('data', (data) => {
+    console.info(data.toString().trim());
+  });
```

Comment:
Why console.info and not console.error ?

Reply:
I had previously used `.error`, but I noticed that pg_dump sometimes prints warnings as stderr, and when that happened, the dump process would stop halfway. Let me work on an improvement here, and I'll provide an update on the PR shortly.

Reply:
In your other PRs I saw you created a more complete checkError function. So once everything is merged, I think it will be the best option.